src/pages/lib/graphqlapi/query-data/q/platform/[platform].mdx

```diff
@@ -5,7 +5,7 @@ export const meta = {
 
 <InlineFilter filters={["js", "react-native"]}>
 
-You can read application data using the Amplify GraphQL client. In this guide we will review the difference between reading data and getting data, how to filter query results to get just the data you need, and how to paginate results to make your data more manageable. We will also show you how to cancel these requests when needed.
+You can read application data using the Amplify GraphQL client. In this guide, we will review the difference between reading data and getting data, how to filter query results to get just the data you need, and how to paginate results to make your data more manageable. We will also show you how to cancel these requests when needed.
 
 Before you begin, you will need:
 
```

Comment:
If Connect guide is not ready - the JS link below will need to be updated to getting started page.